src/api/entities/Identity/index.ts

```diff
@@ -981,4 +986,64 @@ export class Identity extends Entity<UniqueIdentifiers, string> {
 
     return boolToBoolean(rawIsApproved);
   }
+
+  /**
+   * Returns the list of MultiSig accounts linked with this Identity along with the signatories for whom the MultiSig acts as the signer
```

Comment:
```suggestion
   * Returns the list of MultiSig accounts linked with this Identity along with the signatories
```

Maybe its me, but I don't really get the last part here